datashuttle/utils/getters.py

```diff
@@ -339,3 +340,86 @@ def get_all_sub_and_ses_names(
             )
 
     return {"sub": all_sub_folder_names, "ses": all_ses_folder_names}
+
+
+def get_all_sub_and_ses(
```

Comment:
Thanks for this @Transyltooniaa! In a recent PR (due to be merged soon, #464) this functionality was added to `get_all_sub_and_ses_names` which is now called `get_all_sub_and_ses_paths`. This will be merged soon and we can take another look at this. Please feel free to start a discussion on any issues to brainstorm an approach to avoid clashes, apologies for this I still hope coding this up was useful.